styles.css

```diff
@@ -184,22 +182,29 @@ textarea#title {
 
 .show-starred-btn {
   background-color: #A8AAD2;
-  color: #1F1F3C; /* Dark purple */
-  padding: 10px 20px; 
-  border: none; 
-  cursor: pointer; 
-  margin-top: 20px; 
+  color: #1F1F3C;
+  /* Dark purple */
+  padding: 10px 20px;
+  border: none;
+  cursor: pointer;
+  margin-top: 20px;
   margin-left: auto;
   margin-right: auto;
-  text-align: center; 
+  text-align: center;
   width: 90%;
   font-size: large;
   font-weight: 700;
-  }
-  
+}
+
 .show-starred-btn:hover {
-  background-color: grey; 
-  }
+  background-color: grey;
+}
+
+.disabled {
+  background-color: lightgray;
+  color: grey;
+  cursor: not-allowed;
+}
```

Comment:
@ben-rosner-williamsburg I see that you moved this, but it's still not working on load for me... maybe we need to add an event listener for window load to make this disabled like the save button?

Reply:
@ericahagle Yeah it was working before, now it's not. I'm going to try to troubleshoot.

Reply:
Yep! I see it's because of the removal of the faveIdeas array. I say timebox how long you want to spend trying to adjust to that refactor, and if it's too complicated, we can always add the array back in.

Reply:
The additional problem that I'm seeing, is that if we're using the array for the button, we end up with a bug around deleting the idea. Right now, we're only deleting it from the currentIdeas array, so it stays in the faveIdeas array if you delete it while still faved. We can either fix this by finding a solution for the button that doesn't involve the faveIdeas array, or we could just put something in the delete function that removes it from both arrays at the same time, if it was favorited when the user hit the delete button.

Reply:
Here's the requirement from the spec that says we should remove all deleted items from the data model:

When a user clicks the delete button:
- that idea should be removed from the data model.